modules/administration-guide/partials/proc_configuring-github-oauth.adoc

```diff
@@ -16,6 +16,8 @@ OAuth for GitHub allows for automatic SSH key upload to GitHub.
 
 * Create a link:https://developer.github.com/apps/building-oauth-apps/creating-an-oauth-app[OAuth application in GitHub] using {prod-short} URL as the value for the application `Homepage URL` and {identity-provider} GitHub endpoint URL as the value for Authorization callback URL. The default values are `++https://++{prod-deployment}-{prod-namespace}.__<DOMAIN>__/` and `++https://++keycloak-{prod-namespace}.__<DOMAIN>__/auth/realms/{prod-deployment}/broker/github/endpoint` respectively, where `__<DOMAIN>__` is {orch-name} cluster domain.
 
+* When {prod-short} is deployed using native authentication, the value for `Authorization callback URL` should be {prod-short} OAuth callback URL. The default value is `++https://++{prod-deployment}-{prod-namespace}.__<DOMAIN>__/api/oauth/callback`.
```

Comment:
What "native authentication"?

Reply:
```suggestion
* When {prod-short} is deployed using native authentication, set the `Authorization callback URL` value to {prod-short} OAuth callback URL. The default value is `++https://++{prod-deployment}-{prod-namespace}.__<DOMAIN>__/api/oauth/callback`.
```

Reply:
"_When {prod-short} is deployed using native authentication_" - is "_when_" or "_if_" more appropriate here? "_When_" is what happens always; "_if_" is what may happen under certain conditions.

Reply:
It's unclear from this sentence where (in which CRD? or in which .yaml file? ...?) the admin can set "the value for `Authorization callback URL`"?

Reply:
this sentence is an alternative to the sentence above. If the {devworkspace} engine is enabled we should put one callback our, if not  - another.

Reply:
In such case, would it be better to split the second half of the existing paragraph into two alternatives? For example:

_Create a link:https://developer.github.com/apps/building-oauth-apps/creating-an-oauth-app[OAuth application in GitHub] using {prod-short} URL as the value for the application `Homepage URL` and {identity-provider} GitHub endpoint URL as the value for Authorization callback URL:_
* [edit the beginning of the last sentence in the existing paragraph, for example:] _If the {devworkspace} engine is not enabled... The default values are `++https://++{prod-deployment}-{prod-namespace}.__<DOMAIN>__/` and `++https://++keycloak-{prod-namespace}.__<DOMAIN>__/auth/realms/{prod-deployment}/broker/github/endpoint` respectively, where `__<DOMAIN>__` is {orch-name} cluster domain._
* [edit the beginning of the new paragraph, for example:] _If the {devworkspace} engine is not enabled... When {prod-short} is deployed using native authentication, the value for `Authorization callback URL` should be {prod-short} OAuth callback URL. The default value is `++https://++{prod-deployment}-{prod-namespace}.__<DOMAIN>__/api/oauth/callback`._

Reply:
Sounds great. Let me check if I can change the pr.

Reply:
It's not clear to me, what is the current proposal. If we're going to deprecate che-server engine in next release, we should not make much effort to describe everything for all cases. Also I think it's enough to say something like "For devworkspace engine", no need to say the native authentication stuff, because it's now the default with devworkspaces.

Reply:
The docs still have content related to che-server engine, so we need clarity about which content needs to be removed and which content is going to stay.